components/atom/input/src/Input/Features/Addon/index.scss

```diff
@@ -4,6 +4,13 @@ $bgc-atom-input-addon: $c-gray-lightest !default;
 $bdc-atom-input-addon: $c-gray-light !default;
 $c-atom-input-addon: $c-gray-light !default;
 
+$l-atom-input-icon: $gutter !default;
+$w-atom-input-icon: $sz-icon-m !default;
+$h-atom-input-icon: $sz-icon-m !default;
+$t-atom-input-icon: 50% !default;
+$trf-ty-atom-input-icon: -50% !default;
+$pl-atom-input-icon: 50px !default;
```

Comment:
Are you sure this is not `$sz-icon-m*2`?

Reply:
This value uses a `sui-theme` variable and is configurable so I don't see the issue here. What's your point/suggestion?